src/backend/marsha/core/api/playlist.py

```diff
@@ -134,3 +137,26 @@ def destroy(self, request, *args, **kwargs):
             return super().destroy(request, *args, **kwargs)
         except ProtectedError:
             return Response("Resources are still attached to playlist", status=400)
+
+    @action(detail=True, methods=["post"])
+    # pylint: disable=unused-argument
+    def claim(self, request, *args, **kwargs):
+        """Claim a playlist by creating a PlaylistAccess."""
+        playlist = self.get_object()
+        playlist_access_exists = PlaylistAccess.objects.filter(
+            playlist=playlist,
+        ).exists()
+        PlaylistAccess.objects.get_or_create(
+            playlist=playlist,
+            user_id=request.user.id,
+            defaults={
+                "role": INSTRUCTOR if playlist_access_exists else ADMINISTRATOR,
+            },
+        )
+
+        # should we set playlist organization to user organization?
+        # should we set playlist consumer_site to user consumer_site through user_association ?
```

Comment:
Same comment as above. But the playlist should already be in this consumer_site ?

Reply:
> But the playlist should already be in this consumer_site ?

It doesn't seems like.